From cc14f196dfde252adace5f1305b83c84af0084ee Mon Sep 17 00:00:00 2001
From: Ryan Gonzalez <rymg19@gmail.com>
Date: Tue, 17 Nov 2020 22:57:29 -0600
Subject: [PATCH] Import chromium-71.0.3578.98-widevine-r3.patch + arm64
 support

Taken from the Fedora 33 repositories, with an extra change from Ubuntu
to enable it on arm64.
---
 third_party/widevine/cdm/widevine.gni           | 2 +-
 third_party/widevine/cdm/widevine_cdm_version.h | 2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/third_party/widevine/cdm/widevine.gni b/third_party/widevine/cdm/widevine.gni
index 58f073ca562ca..d4219f13cb3cb 100644
--- a/third_party/widevine/cdm/widevine.gni
+++ b/third_party/widevine/cdm/widevine.gni
@@ -27,7 +27,7 @@ if (is_chromeos && !is_chromeos_device) {
 library_widevine_cdm_available =
     (is_chromeos &&
      (target_cpu == "x64" || target_cpu == "arm" || target_cpu == "arm64")) ||
-    (target_os == "linux" && target_cpu == "x64") ||
+    (target_os == "linux") ||
     (target_os == "mac" && (target_cpu == "x64" || target_cpu == "arm64")) ||
     (target_os == "win" &&
      (target_cpu == "x86" || target_cpu == "x64" || target_cpu == "arm64"))
diff --git a/third_party/widevine/cdm/widevine_cdm_version.h b/third_party/widevine/cdm/widevine_cdm_version.h
index db80700b6c1b2..370bb3c9724ec 100644
--- a/third_party/widevine/cdm/widevine_cdm_version.h
+++ b/third_party/widevine/cdm/widevine_cdm_version.h
@@ -12,4 +12,6 @@
 //  - WIDEVINE_CDM_VERSION_STRING (with the version of the CDM that's available
 //    as a string, e.g., "1.0.123.456").
 
+#define WIDEVINE_CDM_VERSION_STRING "unknown"
+
 #endif  // WIDEVINE_CDM_VERSION_H_
-- 
2.43.2

